<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayerForm</class>
 <widget class="QMainWindow" name="VideoPlayerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>326</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Text here</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="view">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>432</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: black;
	color: white;
	font: 11pt &quot;Arial&quot;;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="acceptButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 170, 0);
	border-style: outset;
	border-width: 2px;
	border-color: beige;
	color: white;
	font: 9pt &quot;Arial black&quot;;
	border-radius: 37px;
}

QPushButton:hover{
	background-color: rgb(0, 190, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 117, 0);
	border-color:  rgb(200, 200, 200);
	color: rgb(200, 200, 200);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Принять</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(170, 0, 0);
	border-style: outset;
	border-width: 2px;
	border-color:  rgb(255, 255, 255);
	color: rgb(255, 255, 255);
	font: 9pt &quot;Arial black&quot;;
	border-radius: 37px;
}

QPushButton:hover{
	background-color: rgb(190, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(120, 0, 0);
	border-color:  rgb(200, 200, 200);
	color: rgb(200, 200, 200);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>Отклонить</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>20</width>
      <height>160</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>160</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical {
    background: rgba(255, 255, 255,0);
    position: absolute;
    left: 4px; right: 4px;
}

QSlider::handle:vertical {
	height: 10px;
	background: rgba(0, 130, 0,255);
	border-radius: 2px;
}

QSlider::add-page:vertical {
	background: rgba(0, 150, 0,150);
	border-radius: 5px;
}

QSlider::sub-page:vertical {
	background: rgba(255, 255, 255,30);
	border-radius: 5px;
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="screenBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255,50);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="soundBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255,50);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
